modules/group/variables.tf

```diff
@@ -1,9 +1,17 @@
-variable "group" {
-  description = "Name and path of group"
-  type = object({
-    name     = string
-    path     = string
-    policies = list(string) # policy arns
-  })
-  nullable = false
+variable "name" {
+  description = "The name of the user"
+  type = string
+  default = null
+}
+
+variable "path" {
+  description = "The path of the user"
+  type = string
+  default = null
+}
+
+variable "policies" {
+  description = "The tags of the user"
```

Comment:
not sure if this description is accurate? policies = tags of the user?

Reply:
fixed :)